src/GDAL.jl

```diff
@@ -30,7 +30,8 @@ module C
 end
 
 include("C/misc.jl")
-include("C/common.jl")
+include("types.jl")
+include("common.jl")
```

Comment:
Should the aliases in `common.jl` be considered as part of the (low-level) C interface? My understanding's that

- they also contain a lot of typealiases like `ANONYMOUS_15`
- we might want to reserve `src/common.jl` for our own keywords